### DISCUSSION TITLE
is there support for k3s.

### DISCUSSION THREAD

**Discussion by Akshath-Nagulapally** [kubero-dev/kubero#372] [Q&A] [answered]
Multicloud deployments is what I would like to use kubero for and I was wondering if kubero supports k3s.

**Comment by mms-gianni** [comment, marked as answer]
Sure. I'm using K3s my self on a RaspberryPy for development.

**Comment by ravecat** [comment]
@mms-gianni are you use manual installation? could your share installation steps here?
